docs/imagecustomizer/api/cli.md

```diff
@@ -39,26 +39,22 @@ provided in the configuration file. If both `output.image.path` and
 `--output-image-file` are provided, then the `--output-image-file` value
 is used.
 
-
 The file path to write the final customized image to.
 
 Added in v0.3.
 
 ## --output-image-format=FORMAT
 
-The image format of the the final customized image.
-
-Options: vhd, vhd-fixed, vhdx, qcow2, raw, iso, and [cosi](./cosi.md).
-
-At least one of `--output-image-format` and `--output-split-partitions-format` is
-required.
+Required, unless
+[output.image.format](./configuration/outputImage.md#format-string) is
+provided in the configuration file. If both `output.image.format` and
+`--output-image-format` are provided, then the `--output-image-format` value
+is used.
 
-The vhd-fixed option outputs a fixed size VHD image. This is the required format for
-VMs in Azure.
+The image format of the the final customized image.
 
-When the output image format is set to iso, the generated image is a LiveOS
-iso image. For more details on this format, see:
-[Image Customizer ISO Support](../concepts/iso.md).
+See [output.image.format](./configuration/outputImage.md#format-string)
```

Comment:
Forcing a user to jump between pages to find information is a little annoying for the user. So, I would duplicate the list of supported formats in both docs.